src/Plugin/resource/RateLimitedResource.php

```diff
@@ -196,4 +202,67 @@ public function getVersion() {
     return $this->subject->getVersion();
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function versionedUrl($path = '', $options = array(), $version_string = TRUE) {
```

Comment:
I'm not a big fan of this proxy methods -- so previous comments. While it saves a few key strokes, it makes it unclear who's responsible for what.